CharacterGenerator/MinionMadness/MInionManager.m

```diff
@@ -0,0 +1,126 @@
+//
+//  MInionManager.m
+//  MinionMadness
+//
+//  Created by Jovanny Espinal on 8/8/15.
+//  Copyright (c) 2015 Jovanny Espinal. All rights reserved.
+//
+
+#import "MinionManager.h"
+
+@implementation MinionManager
+
+
++(id)sharedManager{
+    static  MinionManager *sharedMyManager = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        sharedMyManager = [[self alloc] init];
+    });
+    return sharedMyManager;
+}
+
+- (id)init {
+    if (self = [super init]) {
+        self.mutableMinion = [NSMutableArray new];
+        
+        Character *dave = [[Character alloc] init];
```

Comment:
Not sure why you need to have properties about the minions in both MinionManager and QuizData - probably better to only have one central location with the data.